modules/adding-a-custom-model-serving-runtime-for-the-single-model-serving-platform.adoc

```diff
@@ -3,11 +3,11 @@
 [id="adding-a-custom-model-serving-runtime-for-the-single-model-serving-platform_{context}"]
 = Adding a custom model-serving runtime for the single-model serving platform
 ifdef::upstream[]
-A model-serving runtime adds support for a specified set of model frameworks (that is, formats). You have the option of using the link:{odhdocshome}/serving-models/#about-the-single-model-serving-platform_serving-large-models[pre-installed runtimes] included with {productname-short} or adding your own, custom runtimes. This is useful in instances where the pre-installed runtimes don't meet your needs. For example, you might find that the TGIS runtime does not support a particular model format that is supported by link:https://huggingface.co/docs/text-generation-inference/supported_models[Hugging Face Text Generation Inference (TGI)^]. In this case, you can create a custom runtime to add support for the model.
+A model-serving runtime adds support for a specified set of model frameworks and the model formats supported by those frameworks. You can use the link:{odhdocshome}/serving-models/#about-the-single-model-serving-platform_serving-large-models[pre-installed runtimes] included with {productname-short}. You can also add your own custom runtimes if the default runtimes do not meet your needs. For example, if the TGIS runtime does not support a model format that is supported by link:https://huggingface.co/docs/text-generation-inference/supported_models[Hugging Face Text Generation Inference (TGI)^], you can create a custom runtime to add support for the model.
```

Comment:
Suggest the following minor tweak to improve readability:

"You can use the link:{odhdocshome}/serving-models/#about-the-single-model-serving-platform_serving-large-models[pre-installed runtimes] **that are** included with {productname-short}"